src/services/measure.service.js

```diff
@@ -90,9 +90,9 @@ const search = async (args, { req }) => {
  * takes a measureReport and a set of required data with which to calculate the measure and
  * creates new documents for the measureReport and requirements in the appropriate collections.
```

Comment:
```suggestion
 * creates new documents for the measureReport and required data in the appropriate collections.
```

Reply:
These aren't on you, but trying to read this hurt my brain a little, so I changed it. Feel free to reject if you disagree